site/src/main.rs

```diff
@@ -213,6 +217,8 @@ pub struct ButtonStyle {
 pub enum Route {
     #[get("/")]
     Root,
+    #[get("/min-version")]
```

Comment:
possibly move it under /api/min-version, In theory it might make sense in the future to extract api routes into it's own server instance so better manage the traffic, but not a big deal either way